tests/foreman/cli/test_activationkey.py

```diff
@@ -58,173 +56,167 @@ def setUp(self):
 
     def _make_activation_key(self, options=None):
         """Make a new activation key and assert its success"""
-
         if options is None:
             options = {}
 
         # Use default organization if None are provided
-        if (
-                not options.get('organization', None)
-                and not options.get('organization-label', None)
-                and not options.get('organization-id', None)):
+        if (not options.get('organization', None)
+            and not options.get('organization-label', None)
+            and not options.get('organization-id', None)):
             options['organization-id'] = self.org['id']
 
         # Create activation key
         ackey = make_activation_key(options)
 
         # Fetch it
-        result = ActivationKey.info(
-            {
-                'id': ackey['id']
-            }
-        )
+        result = ActivationKey.info({'id': ackey['id']})
 
         self.assertEqual(
             result.return_code,
             0,
-            "Activation key was not found: %s" % str(result.stderr))
+            'Activation key was not found: {}'.format(result.stderr))
         self.assertEqual(
             len(result.stderr),
             0,
-            "No error was expected %s" % str(result.stderr))
+            'No error was expected {}'.format(result.stderr))
 
         # Return the activation key dictionary
         return ackey
 
     @data(
-        {'name': gen_string('alpha', 15)},
-        {'name': gen_string('alphanumeric', 15)},
-        {'name': gen_string('numeric', 15)},
-        {'name': gen_string('latin1', 15)},
-        {'name': gen_string('utf8', 15)},
-        {'name': gen_string('html', 15)},
+        gen_string('alpha'),
+        gen_string('alphanumeric'),
+        gen_string('numeric'),
+        gen_string('latin1'),
+        gen_string('utf8'),
+        gen_string('html'),
     )
-    def test_positive_create_activation_key_1(self, test_data):
+    def test_positive_create_activation_key_1(self, name):
         """@Test: Create Activation key for all variations of
             Activation key name
 
         @Feature: Activation key
 
         @Steps:
+
         1. Create Activation key for all valid Activation Key name variation
 
         @Assert: Activation key is created with chosen name
 
         """
-
-        new_ackey = self._make_activation_key({u'name': test_data['name']})
+        new_ackey_name = self._make_activation_key({u'name': name})['name']
         # Name should match passed data
         self.assertEqual(
-            new_ackey['name'],
-            test_data['name'],
-            ("Names don't match: '%s' != '%s'" %
-             (new_ackey['name'], test_data['name']))
+            new_ackey_name,
+            name,
+            'Names don\'t match: "{}" != "{}"'.format(new_ackey_name, name)
```

Comment:
Is the idea that both items being compared are shown, so given that no information is lost, we might as well add an error message? I think that's the idea. This is definitely a bit of a personal preference thing. I like having more terse code, given that QE folks will typically have to look at code when something fails anyway, so we might as well make the code pretty. But I'm not super set on that opinion, and can certainly live with the error message.